scylla/templates/scylla-config-configmap.yaml

```diff
@@ -120,7 +120,7 @@ data:
     read_request_timeout_in_ms: 5000
 
     # How long the coordinator should wait for writes to complete
-    write_request_timeout_in_ms: 2000
+    write_request_timeout_in_ms: 30000
```

Comment:
Can make this configurable here https://github.com/src-d/charts/blob/master/scylla/values.yaml#L42 ?
This allows us to change it per deployment if needed

Reply:
Fixed